src/FilenameParsing/HashFileIDHelper.php

```diff
@@ -65,6 +66,14 @@ public function lookForVariantIn(ParsedFileID $parsedFileID)
         return  $folder . $this->truncate($parsedFileID->getHash());
     }
 
+    public function getVariantGlob(string $folder, ParsedFileID $parsedFileID): string
+    {
+        $folderWithoutHash = str_replace('/' . $this->truncate($parsedFileID->getHash()), '', $folder);
+        $fileWithoutFolder = preg_replace('#^' . preg_quote($folderWithoutHash, '#') . '/#', '', $parsedFileID->getFilename());
+        $fileWithoutExtension = preg_replace('#\.' . preg_quote(File::get_file_extension($fileWithoutFolder), '#') . '$#', '', $fileWithoutFolder);
+        return $fileWithoutExtension . FileIDHelper::VARIANT_SEPARATOR . '*';
```

Comment:
```suggestion
        $truncatedHash = $this->truncate($parsedFileID->getHash());
        $folderWithoutHash = str_replace('/' . $truncatedHash, '', $folder);
        $folderRx = '#^' . preg_quote($folderWithoutHash, '#') . '/#';
        $fileWithoutFolder = preg_replace($folderRx, '', $parsedFileID->getFilename());
        $extRx = '#\.' . preg_quote(File::get_file_extension($fileWithoutFolder), '#') . '$#';
        $fileWithoutExtension = preg_replace($extRx, '', $fileWithoutFolder);
        return $fileWithoutExtension . FileIDHelper::VARIANT_SEPARATOR . '*';
```
Make it a lot easier to read

Reply:
Done